src/twenty-three-video/twenty-three-video.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable, Logger } from "@nestjs/common";
+import { DisplayApiService } from "../display-api/display-api.service";
+import { HttpService } from "@nestjs/axios";
+import { Slide } from "../display-api/types";
+import { lastValueFrom } from "rxjs";
+import { SlideSlideInput } from "../display-api-client";
+
+interface TwentyThreeVideoListParams {
+  format: string;
+  raw: boolean;
+  tags?: string;
+  album_id?: string;
+}
+
+@Injectable()
+export class TwentyThreeVideoService {
+  constructor(
+    private readonly logger: Logger,
+    private readonly httpService: HttpService,
+    private readonly displayApi: DisplayApiService
+  ) {}
+
+  // TODO, move this out into configuration.
+  private readonly baseUrl = "https://video.kk.dk/api/photo/list";
+  private readonly slideType = "TwentyThreeVideo";
+
+  async syncAllSlides(): Promise<void> {
+    this.logger.debug("TwentyThreeVideoService::syncAllSlides");
+    const slides = await this.displayApi.fetchSlides(this.slideType);
+    for (const slide of slides) {
+      await this.syncSlide(slide);
```

Comment:
Be careful not to allow one bad slide to block subsequent slides from being updated. Consider adding a try/catch here and log + continue in case of an error.

Reply:
Sync slide can't crash, because `fetchTwentyThreeVideoList` and `updateSlide` has try/catch